exercises/G-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+const numberOfStudents=40;
+const numberOfMentors = 8;
+
+console.log(numberOfMentors + numberOfStudents);
```

Comment:
Good one.
Just a few details are missing about your solution. 
Also, a console description of what you are calculating.